<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="ACER" failures="1" tests="1" name="EdgeTest(failed)" time="6.034" errors="0" timestamp="2023-03-27T11:25:18 IST">
  <testcase classname="DemoTestNG.DemoTestNG2" name="VerifyMessageonSignUppage" time="0.026">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Text is wrong expected [Add new user] but found [Selenium Webdriver]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Text is wrong expected [Add new user] but found [Selenium Webdriver]
at DemoTestNG.DemoTestNG2.VerifyMessageonSignUppage(DemoTestNG2.java:102)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- VerifyMessageonSignUppage -->
  <testcase classname="DemoTestNG.DemoTestNG2" name="VerifyMessageonSignUpPageisSignup" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyMessageonSignUpPageisSignup -->
  <testcase classname="DemoTestNG.DemoTestNG2" name="VerifysignupButtononSignupPage" time="0.0">
    <ignored/>
  </testcase> <!-- VerifysignupButtononSignupPage -->
  <testcase classname="DemoTestNG.DemoTestNG2" name="VerifytheaccountisalreadyExit" time="0.0">
    <ignored/>
  </testcase> <!-- VerifytheaccountisalreadyExit -->
</testsuite> <!-- EdgeTest(failed) -->
